modules/ichorcna/1.0/src/runIchorCNA.R

```diff
@@ -0,0 +1,423 @@
+# file:   ichorCNA.R
```

Comment:
This file is now downloaded directly, right?

Reply:
No this one is the one that I had to modify to allow hs37d5 to be included. They hard-coded the available genomes